commands/validate.go

```diff
@@ -0,0 +1,91 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/Masterminds/semver"
+	"github.com/pivotal-cf/jhanda"
+	"github.com/pivotal-cf/kiln/internal/cargo"
+	"gopkg.in/src-d/go-billy.v4"
+	"gopkg.in/src-d/go-billy.v4/osfs"
+	"strings"
+)
+
+type Validate struct {
+	Options struct {
+		Kilnfile       string   `short:"kf" long:"kilnfile" default:"Kilnfile" description:"path to Kilnfile"`
+		VariablesFiles []string `short:"vf" long:"variables-file" description:"path to variables file"`
```

Comment:
**question:** do we have examples of var files?

Reply:
Yes. Kiln fetch uses them for s3 credentials. The README has an example.